src/MusicStore/samples/MusicStore/wwwroot/Scripts/modernizr-2.6.2.js

```diff
@@ -868,7 +868,7 @@ window.Modernizr = (function( window, document, undefined ) {
 
     /*>>webforms*/
     // input features and input types go directly onto the ret object, bypassing the tests loop.
-    // Hold this guy to execute in a moment.
+    // Hold this to execute in a moment.
```

Comment:
Is this an external dependancy? If so, the change should be made there, and left here untouched (versioning).

Reply:
Apologies, I thought I had stuck to this repo when doing a search on solution but must have missed one. I will remove it and update PR.

Reply:
This is updated now. Aswell as making some of the language a bit more passive form as suggested.

Thanks,
Carole